src/include/sof/lib/kcps-budget-manager.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ *
+ * Author: Krzysztof Frydryk <frydryk.krzysztof@intel.com>
+ */
+
+#ifndef __SOF_LIB_CPS_BUDGET_MANAGER_H__
+#define __SOF_LIB_CPS_BUDGET_MANAGER_H__
```

Comment:
The name is a bit challenging, but I can't come up with a better one. "DSP Cycle Budget Manager" would be more explicit, but "CPS" is used in the IPC messages, so maybe this is good enough.